routes/add_event.js

```diff
@@ -8,15 +8,25 @@ module.exports = (req, res) => {
     categories: req.body.categories,
     accessibilityOptions: req.body.accessibilityOptions,
     startTime: req.body.startDate + 'T' + req.body.startTime,
-    endTime: req.body.endDate + 'T' + req.body.endTime,
+    endTime: req.body.startDate + 'T' + req.body.endTime,
     cost: req.body.cost,
-    imageUrl: req.body.imageUrl,
-    en: {
-      name: req.body.name,
-      description: req.body.description
-    }
+    imageUrl: req.body.imageUrl
   };
 
+  if (req.body.name_english) {
+    requestBody.en = {
+      name: req.body.name_english,
+      description: req.body.description_en ? req.body.description_en : ''
```

Comment:
At the moment this will always at least add an empty description field - it'd be better to only add that field at all if there is a description present. I guess you'd have to do that outside the object declaration but yea

Reply:
why does it matter though ?
it's going to add an empty string either way to the event object.

Reply:
k